covsirphy/cleaning/jhu_complement.py

```diff
@@ -162,6 +165,7 @@ def _post_processing(self, df):
                 Columns: Date (pandas.TimeStamp), Confirmed, Infected, Fatal, Recovered (int)
         """
         df = df.astype(np.int64)
+        df = df.loc[df[self.C] > self.max_ignored]
```

Comment:
I think `df.loc[df[self.C] <= self.max_ignored]` should be added to the returned dataframe by `pd.concat().sort_index()` after line170.

Reply:
With 168 line I meant to completely ignore less than max_ignored cases. Essentially to concat them at the end of post processing means that in the first place this change shall not be done at all and thus remove the 168 line, right?

Reply:
Why the records with `confirmed < max_ignored` will be removed?
If necessary, this could be a new issue because this changes the output of `JHUData.records()` for all countries. We are currently remove only `recovered = 0` records.

Reply:
I had tried that initially, because some small values (<max_ignored) led to such negative values and were problematic, but I suppose the pre_processing change shall cover for this now. I will revert this change then!

Reply:
OK, we will discuss this problem in a new issue because this may improve the accuracy of estimation in the 0th phase, but I think this should be handled in `JHUData` or `Scenario` class because this is not a complement.